noxfile.py

```diff
@@ -39,7 +59,9 @@ def notebooks(session):
     session.install("-e", ".[all,dev]", silent=False)
     if PYBOP_SCHEDULED:
         session.run("pip", "install", f"pybamm=={PYBAMM_VERSION}", silent=False)
-    session.run("pytest", "--nbmake", "--examples", "examples/", external=True)
+    session.run(
+        "pytest", "--notebooks", "--nbmake", "examples/", "-n", "auto", external=True
+    )
```

Comment:
```suggestion
    session.run(
        "pytest", "--notebooks", "--nbmake", "examples/", "-n", "auto",
    )
```

This probably doesn't need `external=True`, since the `pytest` installation from inside the venv will be the one that will end up being used.